tests/common/devices/sonic.py

```diff
@@ -2068,6 +2069,40 @@ def show_syslog(self):
         """
         return self.show_and_parse('show syslog')
 
+    def get_acl_counter(self, acl_table_name, acl_rule_name,
+                        timeout=ACL_COUNTERS_UPDATE_INTERVAL_IN_SEC * 3,
+                        interval=ACL_COUNTERS_UPDATE_INTERVAL_IN_SEC):
+        """
+        Read ACL counter of specific ACL table and ACL rule.
+
+        Args:
+            acl_table_name (str): Name of ACL table.
+            acl_rule_name (str): Name of ACL rule.
+            timeout (int): Maximum time (in second) wait for ACL counter available.
+            interval (int): Retry interval (in second) between read ACL counter.
+
+        Return:
+            packets_count (int): count of packets hit the specific ACL rule.
+        """
+        assert timeout >= 0 and interval > 0  # Validate arguments to avoid infinite loop
+        while timeout >= 0:
```

Comment:
How about add `aclshow -c` to clear the counter before reading the counter?

Reply:
Thanks @bingwang-ms! For this `get_acl_counter` function, I think we should return the ACL counter as it is, clear the counter may not be an expected behavior in all test scenario. We can add a separate `clear_acl_counter` function to apply the clear action. I'll update it later in ths PR.